packages/sdk/src/light_push/light_push.ts

```diff
@@ -67,11 +86,40 @@ export class LightPush implements ILightPush {
     this.retryManager.stop();
   }
 
+  private async selectProtocol(
```

Comment:
nit: please, keep `private` below `public`

damn, we need to setup `eslint` for it :D